webapp/src/components/AddMember/AddMemberPage.vue

```diff
@@ -1,132 +1,170 @@
-                    </li>
-                </ul>
+            <div class="right-side">
+              <div class="chat-bubble">
+                <div class="group-name-box">
+                  <input type="text" v-model="groupName" placeholder="Enter username" class="group-name-input"
+                    @keyup.enter="createGroup" />
+                </div>
+                <br/>
+                <!-- Create group button -->
+                <button class="create-group-button" @click="createGroup" :disabled="!groupName.trim()">
+                  Invite user
+                </button>
+                <!-- Link to go back to homepage -->
+                <router-link to="/home" class="back-home-link">Go back to homepage</router-link>
+              </div>
             </div>
+            <img src="@/assets/img/create-grp-photo.gif" alt="Animated GIF" class="animated-gif"/>
+          </div>
         </div>
-        <SCLogo></SCLogo>
+      </div>
+      <!-- End of main content -->
     </div>
-</template>
-
-<script>
-    import { ref } from 'vue';
-    import NavBar from '../NavBar.vue';
-    import SCLogo from '../SCLogo.vue';
-
-    export default {
-        name: 'AddMemberPage',
-        components: {
-            NavBar,
-            SCLogo
-        },
-        setup(){
-            const groups = ref([
-                {
-                    "groupId": "101",
-                    "groupName": "CS4389"
-                },
-                {
-                    "groupId": "102",
-                    "groupName": "Work"
-                },
-                {
-                    "groupId": "103",
-                    "groupName": "Family"
-                },
-                {
-                    "groupId": "104",
-                    "groupName": "Sports"
-                },
-                {
-                    "groupId": "105",
-                    "groupName": "Book Club"
-                },
-                {
-                    "groupId": "106",
-                    "groupName": "CS4389"
-                },
-                {
-                    "groupId": "107",
-                    "groupName": "Work"
-                },
-                {
-                    "groupId": "108",
-                    "groupName": "Family"
-                },
-                {
-                    "groupId": "109",
-                    "groupName": "Sports"
-                },
-                {
-                    "groupId": "1010",
-                    "groupName": "Book Club"
-                }
-            ]);
-
-            return { groups }
+  </template>
+  
+  <script>
+  import NavBar from '../NavBar.vue';
+  
+  export default {
+    name: 'NewGroupPage',
+    components: {
+      NavBar,
+    },
+    data() {
+      return {
+        groupName: ''
+      };
+    },
+    methods: {
+      createGroup() {
+        if (!this.groupName.trim()) {
+          alert('Please enter a username.');
```

Comment:
Typo in alert? Seems like for something with group, but says 'enter a username'

Reply:
fixed typo